gtkmm-plplot/canvas.cpp

```diff
@@ -192,7 +199,7 @@ bool Canvas::on_button_press_event(GdkEventButton *event) {
       region_selection->get_region_selectable() &&
       !shift_pressed) {
       // double-click -> zoom out!
-      if (event->type == GDK_2BUTTON_PRESS) {
+      if (n_press > 1) {
```

Comment:
Would it be useful to set this to exactly 2?
```suggestion
      if (n_press == 2) {
```